mobile/src/people/navigator/tab-bar-top-custom.component.tsx

```diff
@@ -61,22 +61,22 @@ class TabBarTopCustomImpl extends React.Component<TabBarTopProps & TabBarTopCust
 }
 
 //----------------------------------------------------------------------------
-const stateToProps = (state: AppState): TabBarTopCustomProps => {
-
-    const getDepartment = (st: AppState, emp: Optional<Employee>): Optional<Department> => {
-
-        if (!state.organization || !emp) {
-            return null;
-        }
+function getDepartment(state: AppState, employee: Optional<Employee>): Optional<Department> {
```

Comment:
as long as those are functions you can still have them inside stateToProps (although I'd change it to function too :) ) and have access to 'state' through closure.
```
function stateToProps(state:AppState) {
 function getEmployee() {
   return state.organization.employees....
 }
}
```

Reply:
Yes but no.